src/adapters/uniswap-v2/products/pool/metadata/ethereum.protocol-token.json

```diff
@@ -0,0 +1,1002 @@
+{
+  "0x21b8065d10f73EE2e260e5B47D3344d3Ced7596E": {
+    "protocolToken": {
+      "address": "0x21b8065d10f73EE2e260e5B47D3344d3Ced7596E",
+      "name": "Uniswap V2",
+      "symbol": "UNI-V2",
```

Comment:
Do we want to artificially change the name and/or symbol of the protocol token? All of them are called the same.

Reply:
Done